packages/bot-web-ui/src/components/dashboard/dashboard-component/index.scss

```diff
@@ -101,10 +111,32 @@
                     margin-bottom: 0.8rem;
                 }
             }
+
+            &__content-group {
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+                height: calc(100vh - 20rem);
+                margin-top: 1rem;
+
+                &__header {
+                    margin-bottom: 1.3rem;
+                }
+
+                &__description {
+                    max-width: 44rem;
+                    text-align: center;
+                }
+            }
         }
     }
 }
 
+.side_bar_container_none {
```

Comment:
this way of naming is not align with BEM condition 
if this is the styling for sidebar it's better we have a separate css file for it in its folder

Reply:
done ✅